CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## 3.0.0 (Unreleased)
+
+FEATURES:
+* Add storage cloning.
+
+DEPRECATED FEATURES:
+* `requestCheckTimeoutSecs` is removed from `NewConfiguration` function.
+
+IMPROVEMENTS:
+* Every function (mostly) can be controlled through context.
+* Server shutdown does not run powering off when the server cannot be shut down gracefully. Forcing a server to be shut down has to be done manually.
```

Comment:
`manually ` could be misleading. I would suggest `To ungracefully power off a server, the `StopServer` function should be used.`

Reply:
Done